### DISCUSSION TITLE
Sudo prompt on Wayland

### DISCUSSION THREAD

**Discussion by Gnomasz** [lxqt/lxqt#2549] [Q&A] [answered]
Hi,

I have a question regarding the [blog post on Qt 6 and Wayland](https://lxqt-project.org/blog/2024/02/15/qt-6-and-wayland/): the table at the end indicates that LXQt sudo is not planned for Wayland. I'm curious: what will be the alternative to open GUI apps as sudo?

**Comment by stefonarch** [comment, marked as answer]
Afaik you simply can't do it without hacks.

 For file management the "Open tab as admin" in pcmanfm-qt works fine under wayland, for text editing just edit a file and on save the lxqt-policykit dialog will prompt you for the passwd. 
I don't use any other GUI tool with sudo, maybe there are other situations I ignore

**Comment by tsujan** [comment]
In addition to what @stefonarch explained correctly:

`lxsudo` can already be used under Wayland with a simple trick. For example,

```sh
xhost +SI:localuser:root
lxsudo fpad
xhost -SI:localuser:root
```
